draft-ietf-core-dns-over-coap.md

```diff
@@ -391,7 +392,20 @@ A successful response:
       ;; ANSWER SECTION:
       ;example.org.         79689   IN      AAAA    2001:db8:1::1:2:3:4
 
-When a DNS error—NotImp (RCODE = 4) in response to a DNS Update (OPCODE = 5) for "example.org" in this case—is noted in the DNS response, the CoAP response still indicates success.
+When a DNS error—NxDomain (RCODE = 3) for "does.not.exist" in this case—is noted in the DNS response, the CoAP response still indicates success.
+
+    2.05 Content
+    Content-Format: application/dns-message
+    Payload (human-readable):
+    ;; ->>HEADER<<- opcode: QUERY, status: NXDOMAIN, id: 0
+    ;; flags: qr rd ra; QUERY: 1, ANSWER: 0, AUTHORITY: 1, ADDITIONAL: 0
```

Comment:
There 1 record in the Authority section included. But this record is not shown in the human-readable payload format?

For example "dig" seems to do this:
https://stackoverflow.com/questions/16072817/what-does-the-authority-section-mean-in-dig-results

Reply:
Same.